app/models/ems_event.rb

```diff
@@ -338,6 +338,29 @@ def ems_refresh_target
     ext_management_system
   end
 
+  def src_container_replicator
```

Comment:
Instead of adding these methods, relationships should be added into EventStream like [for src_vm](https://github.com/ManageIQ/manageiq/blob/3923a425262528a60fc1d65c3ddb82ebb9f4cac1/app/models/event_stream.rb#L7).